### ISSUE TITLE
[Accessibility] Tutorial: next/previous buttons are not labeled

### ISSUE THREAD

**Issue by shortwavesurfer2009** [KewbitXMR/haveno-plus#5]
When using a screen reader such as TalkBack, the next and previous buttons just say "button" but should say "next" or "previous"

**Comment by KewbitXMR** [comment]
Completed, and agree it does feel a lot better, will close this let me know if you think of any other improvements!
